src/client/scripts/structures/user.js

```diff
@@ -25,24 +26,145 @@ export class User {
       this.metadata = metadata;
    }
 
+   /**
+    * @todo - Uncomment follower parts - as of right now, active user's followers/following aren't in database
```

Comment:
I see what you mean regarding the mock user not having legitimate followers, so it makes sense to comment the function calls out for testing. Once I have a more robust example user ill uncomment those. This looks good to me. I see you are separating the local storage keys into logical groups like followers, following, etc. Is the idea here so that we can have a more frequently updated local storage by calling the internal updateLocal_____() functions? 

The way this looks to me is, every time the user does something which modifies state (ex. update a deck)
we
- update local storage for that category (deck, or user, etc)
- update the database with that information

One thing to keep in mind is, we need to absolutely make sure that establishLocalStorage is called prior to anything else, as otherwise, we are going to be calling localStorage.getItem() on an invalid key because it wasn't initialized yet. 

Overall good here

Reply:
The updateLocal functions were mainly just to use for other user methods like registerDeck or registerFollowing, so you could use those methods without having to worry about local storage being out of date.